Vagrantfile

```diff
@@ -6,72 +6,23 @@
 # backwards compatibility). Please don't change it unless you know what
 # you're doing.
 
+$script = <<SCRIPT
+cd /home/vagrant/oppia
+apt-get update && apt-get install -y python-pip python-dev unzip
+bash ./scripts/setup.sh
+bash ./scripts/install_third_party.sh
+SCRIPT
```

Comment:
I would say yes... or we could do it in a way that just asks once 

(either pulling it all to one `apt-get` line or by manually prompting via bash,  something akin to "I'm going to install all of those packages,  alright?" 

@seanlip any concerns about auto approving a package install?